robot/Crisis-Management/tests/browser/t3_availability_records_created.robot

```diff
@@ -0,0 +1,49 @@
+*** Settings ***
+
+Resource        robot/Crisis-Management/resources/CrisisManagement.robot
+Library
+...             cumulusci.robotframework.PageObjects
+...             robot/Crisis-Management/resources/ContactPageObject.py
+...             robot/Crisis-Management/resources/CrisisManagement.py
+
+Suite Setup     Run keywords
+...             Open Test Browser
+
+Suite Teardown  Delete Records and Close Browser
+
+*** Variables ***
+${Last_name}   tractionuser
+${status1}    On staff
+${status2}    Not Available
+
+
+*** Test Cases ***
+
+1. Create a new staff with status equals on staff
+   # Login as admin and create a new staff wit status Not Available
+    Go To Page                                           Listing                                 Contact
+    Click Object Button                                  New
+    Wait For Modal                                       New                                     Staff
+    Populate Modal Form
+    ...                                                  Last Name=${Last_name}
+
+    Populate Lookup Field                                Account Name                            Robot Hospital
+    Select Value From Dropdown                           Status                                  ${status1}
```

Comment:
why use only last name? usually any contact would be created with botth first and last name so curious. Also, did using populate modal form not work for populating lookup field and dropdown?

Reply:
Lastname was the only mandatory field, but added the firstname too and made couple of changes